pkg/network/protocols/http/telemetry.go

```diff
@@ -8,14 +8,29 @@
 package http
 
 import (
+	"strconv"
 	"time"
 
+	"go.uber.org/atomic"
+
 	libtelemetry "github.com/DataDog/datadog-agent/pkg/network/protocols/telemetry"
+	pkgtelemetry "github.com/DataDog/datadog-agent/pkg/telemetry"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
-	"go.uber.org/atomic"
 )
 
+var telem = struct {
+	hits, dropped, rejected, malformed pkgtelemetry.Counter
+	aggregations                       pkgtelemetry.Counter
+}{
+	pkgtelemetry.NewCounter("usm__http", "hits", []string{"protocol", "status_class"}, ""),
+	pkgtelemetry.NewCounter("usm__http", "dropped", []string{"protocol"}, ""),
+	pkgtelemetry.NewCounter("usm__http", "rejected", []string{"protocol"}, ""),
+	pkgtelemetry.NewCounter("usm__http", "malformed", []string{"protocol"}, ""),
+	pkgtelemetry.NewCounter("usm__http", "aggregations", []string{"protocol"}, ""),
+}
+
 type Telemetry struct {
+	proto                                       string
 	LastCheck                                   *atomic.Int64
 	hits1XX, hits2XX, hits3XX, hits4XX, hits5XX *libtelemetry.Metric
```

Comment:
Why are these a part of the `libtelemetry` and the other metrics are `pkgtelemetry` counter telemetry?